lib/venue.rb

```diff
@@ -1,2 +1,34 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+  def initialize(name, capacity)
+    @name     = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(patron)
+    @patrons << patron
+  end
+
+  def yell_at_patrons
+    @patrons.map do |patron|
+      patron.upcase
+    end
+  end
+
+  def over_capacity?
```

Comment:
This is great! In fact, this actually works without the `if` block, since `@patrons.count > capacity` returns `true` or `false`. Try removing the `if` on line 22 and commenting out lines 23-26 to see for yourself.